src/components/header/NavMenu.js

```diff
@@ -0,0 +1,61 @@
+import React from 'react';
+import { useState } from 'react';
+import { Link } from 'react-router-dom';
+
+const NavMenu = ({
+  id,
+  name,
+  arrowClass,
+  arrowIcon,
+  listMan,
+  listFemale,
+  listAll,
+}) => {
+  const [btnHandler, setbtnHandler] = useState(false);
+
+  const btnDisplayHandler = () => {
+    setbtnHandler(!btnHandler);
```

Comment:
```suggestion
    setbtnHandler(prev => !prev);
```
이전값을 이용할때는 함수형 업데이트를 사용하는 것이 좀 더 안전합니다!